### PR TITLE
Fix #1082 Migrate article: Reacting to high load #1082

### LINKED ISSUE #1082: Migrate article: Reacting to high load
Migrate the content from this article to fit the documentation portal format https://help.aiven.io/en/articles/4660372

(Assign this issue to yourself before you start, and remember to mention the issue number when you open your pull request)

### PR DESCRIPTION
**Pull request by sluhur**
# What changed, and why it matters

Fix #1082

### CODE REVIEW

File: _toc.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ entries:
       - file: docs/platform/concepts/disaster-recovery-test-scenarios
       - file: docs/platform/concepts/choosing-timeseries-database
       - file: docs/platform/concepts/service-level-agreement
-      - file: docs/platform/concepts/maintenance-window
```
Comment (ftisiot) [inline _toc.yml:36]: Why is maintenance window gone?

**Review by ftisiot — CHANGES_REQUESTED** [review]
Hi @sluhur, thanks for this! 

I believe we need to check in depth what makes sense to be ported in devportal and how an article fits the [contributing guide](https://github.com/aiven/devportal/blob/main/CONTRIBUTING.rst) 

I believe this is valuable info, but how's it different from #1292?  What type of content should this be? a concept/reference/howto?

Also please take some time to port the format correctly to the devportal style, I currently see:

* Notes that are not devportal notes style
* Links with the link full text and not a caption
* Reference to support which is unnecessary

Doing some of this thinking/work upfront during the porting would mean providing an article which is closer to be merged to devportal and therefore needing less interaction from a review point of view!

**Comment by ftisiot** [comment]
Hi @sluhur, are you planning to work on this or should I close it?

**Comment by lornajane** [comment]
Closing due to inactivity; issue unassigned

**Pull request closed without merging**